************* Module pset_4
pset_4/__init__.py:1:0: C0111: Missing module docstring (missing-docstring)
************* Module pset_4.cli
pset_4/cli.py:12:0: C0303: Trailing whitespace (trailing-whitespace)
pset_4/cli.py:13:0: W0312: Found indentation with tabs instead of spaces (mixed-indentation)
pset_4/cli.py:14:0: W0312: Found indentation with tabs instead of spaces (mixed-indentation)
pset_4/cli.py:14:67: C0326: No space allowed around keyword argument assignment
	main_arg_parser.add_argument("--content-image", type=str, default = '')
                                                                   ^ (bad-whitespace)
pset_4/cli.py:15:0: W0312: Found indentation with tabs instead of spaces (mixed-indentation)
pset_4/cli.py:15:69: C0326: No space allowed around keyword argument assignment
	main_arg_parser.add_argument("--content-scale", type=float, default = 1.0)
                                                                     ^ (bad-whitespace)
pset_4/cli.py:16:0: W0312: Found indentation with tabs instead of spaces (mixed-indentation)
pset_4/cli.py:16:66: C0326: No space allowed around keyword argument assignment
	main_arg_parser.add_argument("--output-image", type=str, default = '')
                                                                  ^ (bad-whitespace)
pset_4/cli.py:17:0: W0312: Found indentation with tabs instead of spaces (mixed-indentation)
pset_4/cli.py:17:58: C0326: No space allowed around keyword argument assignment
	main_arg_parser.add_argument("--cuda", type=int, default = 0)
                                                          ^ (bad-whitespace)
pset_4/cli.py:18:0: W0312: Found indentation with tabs instead of spaces (mixed-indentation)
pset_4/cli.py:18:65: C0326: No space allowed around keyword argument assignment
	main_arg_parser.add_argument("--export_onnx", type=str, default = '')
                                                                 ^ (bad-whitespace)
pset_4/cli.py:19:0: W0312: Found indentation with tabs instead of spaces (mixed-indentation)
pset_4/cli.py:20:0: W0312: Found indentation with tabs instead of spaces (mixed-indentation)
pset_4/cli.py:22:0: W0312: Found indentation with tabs instead of spaces (mixed-indentation)
pset_4/cli.py:24:0: W0312: Found indentation with tabs instead of spaces (mixed-indentation)
pset_4/cli.py:25:0: C0330: Wrong hanging indentation (remove 4 spaces).
		Stylize(image=args.image, model=args.model)
            |   ^ (bad-continuation)
pset_4/cli.py:28:26: C0303: Trailing whitespace (trailing-whitespace)
pset_4/cli.py:29:0: W0312: Found indentation with tabs instead of spaces (mixed-indentation)
pset_4/cli.py:7:0: E0401: Unable to import 'luigi' (import-error)
pset_4/cli.py:11:0: C0111: Missing function docstring (missing-docstring)
pset_4/cli.py:25:2: E1123: Unexpected keyword argument 'image' in constructor call (unexpected-keyword-arg)
pset_4/cli.py:25:2: E1123: Unexpected keyword argument 'model' in constructor call (unexpected-keyword-arg)
************* Module pset_4.tasks.data
pset_4/tasks/data.py:21:5: C0303: Trailing whitespace (trailing-whitespace)
pset_4/tasks/data.py:25:80: C0303: Trailing whitespace (trailing-whitespace)
pset_4/tasks/data.py:25:0: W0312: Found indentation with tabs instead of spaces (mixed-indentation)
pset_4/tasks/data.py:28:0: W0312: Found indentation with tabs instead of spaces (mixed-indentation)
pset_4/tasks/data.py:30:0: W0312: Found indentation with tabs instead of spaces (mixed-indentation)
pset_4/tasks/data.py:32:0: W0312: Found indentation with tabs instead of spaces (mixed-indentation)
pset_4/tasks/data.py:34:0: W0312: Found indentation with tabs instead of spaces (mixed-indentation)
pset_4/tasks/data.py:35:0: W0312: Found indentation with tabs instead of spaces (mixed-indentation)
pset_4/tasks/data.py:36:0: C0303: Trailing whitespace (trailing-whitespace)
pset_4/tasks/data.py:37:0: W0312: Found indentation with tabs instead of spaces (mixed-indentation)
pset_4/tasks/data.py:40:0: W0312: Found indentation with tabs instead of spaces (mixed-indentation)
pset_4/tasks/data.py:43:0: W0312: Found indentation with tabs instead of spaces (mixed-indentation)
pset_4/tasks/data.py:45:0: W0312: Found indentation with tabs instead of spaces (mixed-indentation)
pset_4/tasks/data.py:47:0: W0312: Found indentation with tabs instead of spaces (mixed-indentation)
pset_4/tasks/data.py:49:0: W0312: Found indentation with tabs instead of spaces (mixed-indentation)
pset_4/tasks/data.py:50:0: W0312: Found indentation with tabs instead of spaces (mixed-indentation)
pset_4/tasks/data.py:52:0: W0312: Found indentation with tabs instead of spaces (mixed-indentation)
pset_4/tasks/data.py:55:0: W0312: Found indentation with tabs instead of spaces (mixed-indentation)
pset_4/tasks/data.py:58:0: W0312: Found indentation with tabs instead of spaces (mixed-indentation)
pset_4/tasks/data.py:59:0: C0303: Trailing whitespace (trailing-whitespace)
pset_4/tasks/data.py:60:0: W0312: Found indentation with tabs instead of spaces (mixed-indentation)
pset_4/tasks/data.py:62:0: W0312: Found indentation with tabs instead of spaces (mixed-indentation)
pset_4/tasks/data.py:63:0: W0312: Found indentation with tabs instead of spaces (mixed-indentation)
pset_4/tasks/data.py:66:0: W0312: Found indentation with tabs instead of spaces (mixed-indentation)
pset_4/tasks/data.py:68:0: W0312: Found indentation with tabs instead of spaces (mixed-indentation)
pset_4/tasks/data.py:69:0: W0312: Found indentation with tabs instead of spaces (mixed-indentation)
pset_4/tasks/data.py:71:0: W0312: Found indentation with tabs instead of spaces (mixed-indentation)
pset_4/tasks/data.py:72:0: W0312: Found indentation with tabs instead of spaces (mixed-indentation)
pset_4/tasks/data.py:73:0: W0312: Found indentation with tabs instead of spaces (mixed-indentation)
pset_4/tasks/data.py:74:0: W0312: Found indentation with tabs instead of spaces (mixed-indentation)
pset_4/tasks/data.py:76:0: W0312: Found indentation with tabs instead of spaces (mixed-indentation)
pset_4/tasks/data.py:77:0: W0312: Found indentation with tabs instead of spaces (mixed-indentation)
pset_4/tasks/data.py:79:0: W0312: Found indentation with tabs instead of spaces (mixed-indentation)
pset_4/tasks/data.py:80:0: W0312: Found indentation with tabs instead of spaces (mixed-indentation)
pset_4/tasks/data.py:82:0: W0312: Found indentation with tabs instead of spaces (mixed-indentation)
pset_4/tasks/data.py:84:0: W0312: Found indentation with tabs instead of spaces (mixed-indentation)
pset_4/tasks/data.py:85:0: W0312: Found indentation with tabs instead of spaces (mixed-indentation)
pset_4/tasks/data.py:87:0: W0312: Found indentation with tabs instead of spaces (mixed-indentation)
pset_4/tasks/data.py:87:23: C0326: No space allowed before bracket
			myS3 = boto3.client ('s3')
                       ^ (bad-whitespace)
pset_4/tasks/data.py:88:0: W0312: Found indentation with tabs instead of spaces (mixed-indentation)
pset_4/tasks/data.py:88:32: C0326: Exactly one space required after comma
			myS3.download_file(src_bucket,src_key,toLocal)
                                ^ (bad-whitespace)
pset_4/tasks/data.py:88:40: C0326: Exactly one space required after comma
			myS3.download_file(src_bucket,src_key,toLocal)
                                        ^ (bad-whitespace)
pset_4/tasks/data.py:90:0: W0312: Found indentation with tabs instead of spaces (mixed-indentation)
pset_4/tasks/data.py:91:0: W0312: Found indentation with tabs instead of spaces (mixed-indentation)
pset_4/tasks/data.py:94:0: W0312: Found indentation with tabs instead of spaces (mixed-indentation)
pset_4/tasks/data.py:97:0: W0312: Found indentation with tabs instead of spaces (mixed-indentation)
pset_4/tasks/data.py:98:0: C0303: Trailing whitespace (trailing-whitespace)
pset_4/tasks/data.py:99:0: W0312: Found indentation with tabs instead of spaces (mixed-indentation)
pset_4/tasks/data.py:101:0: W0312: Found indentation with tabs instead of spaces (mixed-indentation)
pset_4/tasks/data.py:102:0: W0312: Found indentation with tabs instead of spaces (mixed-indentation)
pset_4/tasks/data.py:105:0: W0312: Found indentation with tabs instead of spaces (mixed-indentation)
pset_4/tasks/data.py:107:0: W0312: Found indentation with tabs instead of spaces (mixed-indentation)
pset_4/tasks/data.py:108:0: W0312: Found indentation with tabs instead of spaces (mixed-indentation)
pset_4/tasks/data.py:110:0: W0312: Found indentation with tabs instead of spaces (mixed-indentation)
pset_4/tasks/data.py:111:0: W0312: Found indentation with tabs instead of spaces (mixed-indentation)
pset_4/tasks/data.py:112:0: W0312: Found indentation with tabs instead of spaces (mixed-indentation)
pset_4/tasks/data.py:113:0: W0312: Found indentation with tabs instead of spaces (mixed-indentation)
pset_4/tasks/data.py:115:0: W0312: Found indentation with tabs instead of spaces (mixed-indentation)
pset_4/tasks/data.py:116:0: W0312: Found indentation with tabs instead of spaces (mixed-indentation)
pset_4/tasks/data.py:118:0: W0312: Found indentation with tabs instead of spaces (mixed-indentation)
pset_4/tasks/data.py:119:0: W0312: Found indentation with tabs instead of spaces (mixed-indentation)
pset_4/tasks/data.py:121:0: W0312: Found indentation with tabs instead of spaces (mixed-indentation)
pset_4/tasks/data.py:123:0: W0312: Found indentation with tabs instead of spaces (mixed-indentation)
pset_4/tasks/data.py:124:0: W0312: Found indentation with tabs instead of spaces (mixed-indentation)
pset_4/tasks/data.py:126:0: W0312: Found indentation with tabs instead of spaces (mixed-indentation)
pset_4/tasks/data.py:126:23: C0326: No space allowed before bracket
			myS3 = boto3.client ('s3')
                       ^ (bad-whitespace)
pset_4/tasks/data.py:127:0: W0312: Found indentation with tabs instead of spaces (mixed-indentation)
pset_4/tasks/data.py:127:32: C0326: Exactly one space required after comma
			myS3.download_file(src_bucket,src_key,toLocal)
                                ^ (bad-whitespace)
pset_4/tasks/data.py:127:40: C0326: Exactly one space required after comma
			myS3.download_file(src_bucket,src_key,toLocal)
                                        ^ (bad-whitespace)
pset_4/tasks/data.py:129:0: W0312: Found indentation with tabs instead of spaces (mixed-indentation)
pset_4/tasks/data.py:130:0: W0312: Found indentation with tabs instead of spaces (mixed-indentation)
pset_4/tasks/data.py:134:26: C0303: Trailing whitespace (trailing-whitespace)
pset_4/tasks/data.py:135:0: C0304: Final newline missing (missing-final-newline)
pset_4/tasks/data.py:135:0: W0312: Found indentation with tabs instead of spaces (mixed-indentation)
pset_4/tasks/data.py:18:0: E0401: Unable to import 'luigi' (import-error)
pset_4/tasks/data.py:19:0: E0401: Unable to import 'luigi' (import-error)
pset_4/tasks/data.py:20:0: E0401: Unable to import 'luigi.contrib.s3' (import-error)
pset_4/tasks/data.py:22:0: E0401: Unable to import 'boto3' (import-error)
pset_4/tasks/data.py:32:1: C0111: Missing method docstring (missing-docstring)
pset_4/tasks/data.py:35:2: C0103: Variable name "S3Targetimage" doesn't conform to snake_case naming style (invalid-name)
pset_4/tasks/data.py:24:0: R0903: Too few public methods (1/2) (too-few-public-methods)
pset_4/tasks/data.py:47:1: C0111: Missing method docstring (missing-docstring)
pset_4/tasks/data.py:50:2: C0103: Variable name "S3Targetmodel" doesn't conform to snake_case naming style (invalid-name)
pset_4/tasks/data.py:39:0: R0903: Too few public methods (1/2) (too-few-public-methods)
pset_4/tasks/data.py:62:1: R0201: Method could be a function (no-self-use)
pset_4/tasks/data.py:72:2: C0103: Variable name "LocalTargetmodel" doesn't conform to snake_case naming style (invalid-name)
pset_4/tasks/data.py:79:2: C0103: Variable name "fromS3" doesn't conform to snake_case naming style (invalid-name)
pset_4/tasks/data.py:80:2: C0103: Variable name "toLocal" doesn't conform to snake_case naming style (invalid-name)
pset_4/tasks/data.py:90:9: W0703: Catching too general exception Exception (broad-except)
pset_4/tasks/data.py:84:3: C0103: Variable name "myS3Client" doesn't conform to snake_case naming style (invalid-name)
pset_4/tasks/data.py:85:25: W0212: Access to a protected member _path_to_bucket_and_key of a client class (protected-access)
pset_4/tasks/data.py:87:3: C0103: Variable name "myS3" doesn't conform to snake_case naming style (invalid-name)
pset_4/tasks/data.py:101:1: R0201: Method could be a function (no-self-use)
pset_4/tasks/data.py:111:2: C0103: Variable name "LocalTargetimage" doesn't conform to snake_case naming style (invalid-name)
pset_4/tasks/data.py:118:2: C0103: Variable name "fromS3" doesn't conform to snake_case naming style (invalid-name)
pset_4/tasks/data.py:119:2: C0103: Variable name "toLocal" doesn't conform to snake_case naming style (invalid-name)
pset_4/tasks/data.py:129:9: W0703: Catching too general exception Exception (broad-except)
pset_4/tasks/data.py:123:3: C0103: Variable name "myS3Client" doesn't conform to snake_case naming style (invalid-name)
pset_4/tasks/data.py:124:25: W0212: Access to a protected member _path_to_bucket_and_key of a client class (protected-access)
pset_4/tasks/data.py:126:3: C0103: Variable name "myS3" doesn't conform to snake_case naming style (invalid-name)
************* Module pset_4.tasks.stylize
pset_4/tasks/stylize.py:21:0: C0303: Trailing whitespace (trailing-whitespace)
pset_4/tasks/stylize.py:22:0: W0312: Found indentation with tabs instead of spaces (mixed-indentation)
pset_4/tasks/stylize.py:23:0: W0312: Found indentation with tabs instead of spaces (mixed-indentation)
pset_4/tasks/stylize.py:25:0: W0312: Found indentation with tabs instead of spaces (mixed-indentation)
pset_4/tasks/stylize.py:26:0: W0312: Found indentation with tabs instead of spaces (mixed-indentation)
pset_4/tasks/stylize.py:28:0: W0312: Found indentation with tabs instead of spaces (mixed-indentation)
pset_4/tasks/stylize.py:29:0: W0312: Found indentation with tabs instead of spaces (mixed-indentation)
pset_4/tasks/stylize.py:31:0: W0312: Found indentation with tabs instead of spaces (mixed-indentation)
pset_4/tasks/stylize.py:32:0: C0330: Wrong hanging indentation (remove 4 spaces).
			'image': CopyS3ImageLocally(),
                    |   ^ (bad-continuation)
pset_4/tasks/stylize.py:33:0: C0330: Wrong hanging indentation (remove 4 spaces).
			'model': CopyS3ModelLocally()
                    |   ^ (bad-continuation)
pset_4/tasks/stylize.py:36:0: W0312: Found indentation with tabs instead of spaces (mixed-indentation)
pset_4/tasks/stylize.py:38:0: W0312: Found indentation with tabs instead of spaces (mixed-indentation)
pset_4/tasks/stylize.py:39:0: W0312: Found indentation with tabs instead of spaces (mixed-indentation)
pset_4/tasks/stylize.py:40:0: W0312: Found indentation with tabs instead of spaces (mixed-indentation)
pset_4/tasks/stylize.py:42:0: W0312: Found indentation with tabs instead of spaces (mixed-indentation)
pset_4/tasks/stylize.py:44:0: W0312: Found indentation with tabs instead of spaces (mixed-indentation)
pset_4/tasks/stylize.py:45:0: W0312: Found indentation with tabs instead of spaces (mixed-indentation)
pset_4/tasks/stylize.py:47:0: W0312: Found indentation with tabs instead of spaces (mixed-indentation)
pset_4/tasks/stylize.py:48:0: W0312: Found indentation with tabs instead of spaces (mixed-indentation)
pset_4/tasks/stylize.py:49:0: W0312: Found indentation with tabs instead of spaces (mixed-indentation)
pset_4/tasks/stylize.py:50:0: W0312: Found indentation with tabs instead of spaces (mixed-indentation)
pset_4/tasks/stylize.py:52:0: W0312: Found indentation with tabs instead of spaces (mixed-indentation)
pset_4/tasks/stylize.py:56:0: C0304: Final newline missing (missing-final-newline)
pset_4/tasks/stylize.py:56:0: W0312: Found indentation with tabs instead of spaces (mixed-indentation)
pset_4/tasks/stylize.py:13:0: E0401: Unable to import 'luigi' (import-error)
pset_4/tasks/stylize.py:14:0: E0401: Unable to import 'luigi' (import-error)
pset_4/tasks/stylize.py:18:0: E0401: Unable to import 'pset_4.tasks.tasks.data' (import-error)
pset_4/tasks/stylize.py:20:0: C0111: Missing class docstring (missing-docstring)
pset_4/tasks/stylize.py:28:1: R0201: Method could be a function (no-self-use)
pset_4/tasks/stylize.py:36:1: C0111: Missing method docstring (missing-docstring)
pset_4/tasks/stylize.py:38:33: E0602: Undefined variable 'args' (undefined-variable)
pset_4/tasks/stylize.py:39:2: C0103: Variable name "LocalTargetSImg" doesn't conform to snake_case naming style (invalid-name)
pset_4/tasks/stylize.py:42:1: C0111: Missing method docstring (missing-docstring)
pset_4/tasks/stylize.py:47:3: E0602: Undefined variable 'args' (undefined-variable)
pset_4/tasks/stylize.py:48:3: E0602: Undefined variable 'args' (undefined-variable)
pset_4/tasks/stylize.py:49:3: E0602: Undefined variable 'args' (undefined-variable)
pset_4/tasks/stylize.py:50:3: E0602: Undefined variable 'args' (undefined-variable)
pset_4/tasks/stylize.py:52:11: E0602: Undefined variable 'args' (undefined-variable)
pset_4/tasks/stylize.py:10:0: W0611: Unused import argparse (unused-import)
pset_4/tasks/stylize.py:13:0: W0611: Unused import luigi (unused-import)
************* Module pset_4.tasks
pset_4/tasks/__init__.py:1:0: C0111: Missing module docstring (missing-docstring)

----------------------------------------------------------------------
Your code has been rated at -10.09/10 (previous run: -10.09/10, +0.00)

